src/database/models/userArticleLink.js

```diff
@@ -69,6 +67,10 @@ class UserArticleLink extends Base {
     return this.findOneAndUpdate({ userId, articleId }, data, setOnInsert);
   }
 
+  static async findByUserId(userId) {
```

Comment:
How about we move `find()` into `src/database/models/base.js` so that all models can use `find()` just like they already have `findOneAndUpdate`. As a result, `findByUserId()` is for listing user-article links in UserArticleLink, and `find()` is for general use in all models.

Reply:
But mongodb already have a method named `find()`. I don't see the point extracting that part of logic, looks like a bit over engineered

Reply:
Agree adding `find()` to `Base` so that `Base` API is more consistent 🙆‍♀️